### ISSUE TITLE
Issue Enabling Live Indexing

### ISSUE THREAD

**Issue by JoshDi** [metabrainz/musicbrainz-docker#157]
As of the latest commit, 3164d30, I cannot enable live indexing:

```
root@HTPC-Xeon:/storage/musicbrainz-docker# admin/configure rm replication-cron
Successfully set/updated COMPOSE_FILE in '/storage/musicbrainz-docker/.env'.
root@HTPC-Xeon:/storage/musicbrainz-docker# docker-compose up -d
musicbrainz-docker_db_1 is up-to-date
musicbrainz-docker_redis_1 is up-to-date
musicbrainz-docker_search_1 is up-to-date
musicbrainz-docker_mq_1 is up-to-date
musicbrainz-docker_indexer_1 is up-to-date
Recreating musicbrainz-docker_musicbrainz_1 ... done
root@HTPC-Xeon:/storage/musicbrainz-docker# docker-compose exec indexer python -m sir amqp_setup
2020-06-02 15:07:47,861: Connecting to RabbitMQ
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/local/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/code/sir/__main__.py", line 127, in <module>
    main()
  File "/code/sir/__main__.py", line 123, in main
    func(args)
  File "sir/amqp/setup.py", line 22, in setup_rabbitmq
    conn = util.create_amqp_connection()
  File "sir/util.py", line 138, in create_amqp_connection
    virtual_host=cget("vhost"),
  File "/usr/local/lib/python2.7/site-packages/amqp/connection.py", line 180, in __init__
    (10, 30),  # tune
  File "/usr/local/lib/python2.7/site-packages/amqp/abstract_channel.py", line 67, in wait
    self.channel_id, allowed_methods, timeout)
  File "/usr/local/lib/python2.7/site-packages/amqp/connection.py", line 241, in _wait_method
    channel, method_sig, args, content = read_timeout(timeout)
  File "/usr/local/lib/python2.7/site-packages/amqp/connection.py", line 330, in read_timeout
    return self.method_reader.read_method()
  File "/usr/local/lib/python2.7/site-packages/amqp/method_framing.py", line 189, in read_method
    raise m
socket.error: [Errno 104] Connection reset by peer
```

**Comment by JoshDi** [comment]
Does this have to do with this commit? [Fix amqp_setup after amqp 2.5.2 upgrade in dbfe94a](https://github.com/metabrainz/sir/commit/0424dc5659427a6d8fd4645e442800c1178d1ebf)

If I add **channel.open()** to the **sir/amqp/setup.py** file, I still get an error. I also deleted the .pyc files in the same directory. 

```
root@HTPC-Xeon:/storage/musicbrainz-docker/admin# docker-compose exec indexer bash
root@e09866444883:/code# python -m sir amqp_setup
2020-06-02 15:40:23,949: Connecting to RabbitMQ
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/local/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/code/sir/__main__.py", line 127, in <module>
    main()
  File "/code/sir/__main__.py", line 123, in main
    func(args)
  File "sir/amqp/setup.py", line 22, in setup_rabbitmq
    conn = util.create_amqp_connection()
  File "sir/util.py", line 138, in create_amqp_connection
    virtual_host=cget("vhost"),
  File "/usr/local/lib/python2.7/site-packages/amqp/connection.py", line 180, in __init__
    (10, 30),  # tune
  File "/usr/local/lib/python2.7/site-packages/amqp/abstract_channel.py", line 67, in wait
    self.channel_id, allowed_methods, timeout)
  File "/usr/local/lib/python2.7/site-packages/amqp/connection.py", line 241, in _wait_method
    channel, method_sig, args, content = read_timeout(timeout)
  File "/usr/local/lib/python2.7/site-packages/amqp/connection.py", line 330, in read_timeout
    return self.method_reader.read_method()
  File "/usr/local/lib/python2.7/site-packages/amqp/method_framing.py", line 189, in read_method
    raise m
socket.error: [Errno 104] Connection reset by peer
```

**Comment by JoshDi** [comment]
I ran the following commands and it fixed the problem:

```
root@HTPC-Xeon:/storage/musicbrainz-docker# docker-compose exec mq rabbitmqadmin list vhosts
+------+----------+
| name | messages |
+------+----------+
| /    |          |
+------+----------+
root@HTPC-Xeon:/storage/musicbrainz-docker# docker-compose up --force-recreate -d mq
Recreating musicbrainz-docker_mq_1 ... done
root@HTPC-Xeon:/storage/musicbrainz-docker# docker-compose up --force-recreate -d indexer
Recreating musicbrainz-docker_mq_1     ... done
Recreating musicbrainz-docker_search_1 ... done
Recreating musicbrainz-docker_db_1     ... done
Recreating musicbrainz-docker_indexer_1 ... done
root@HTPC-Xeon:/storage/musicbrainz-docker# docker-compose exec indexer python -m sir amqp_setup
2020-06-02 15:46:21,011: Connecting to RabbitMQ
2020-06-02 15:46:21,020: Declaring exchanes
2020-06-02 15:46:21,024: Declaring queues
2020-06-02 15:46:21,031: Binding queues
2020-06-02 15:46:21,036: Done
root@HTPC-Xeon:/storage/musicbrainz-docker# docker-compose exec mq rabbitmqadmin list vhosts
+-------------------------+----------+
|          name           | messages |
+-------------------------+----------+
| /                       |          |
| /search-index-rebuilder | 0        |
+-------------------------+----------+
```
